.dockerignore

```diff
@@ -0,0 +1,5 @@
+.idea
+.git
+bin
+tests
+api.log
```

Comment:
Should we keep a separate instance of logs on dev machine?

Reply:
I am just not packing developer machine logs to LIVE container